docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+
+Containers in pods are created with endpoints. Other applications communicate with pods by addressing those endpoints. If you specify a container network without a name in a pod definition, it will be assigned to this default network.
+
+In your pod definition you can declare a `host` or `container` network type. Pods created with `host` type share the network namespace of the host. Pods created with `container` type use virtual networking. If you specify the `container` network type, you must also declare a virtual network name in the `name` field. See the [Examples](link) section for the full JSON.
+
+## Ephemeral Storage
+Containers within a pod share ephemeral storage. You can mount volumes by different names on each container in the pod.
+
+## Pod Definitions
+Pods are configured via a JSON pod definition, which is similar to an [application definition](http://mesosphere.github.io/marathon/docs/application-basics.html). You must declare the resources required by each container in the pod, even if Mesos is isolating at a higher (pod) level.  See the [Examples](link) section for complete pod definitions.
+
+### Secrets
+
+Specify a secret in the `secrets` field of your pod definition. The argument should be the fully qualified path to the secret in the store.
+
+```
+{
+	"secrets": /fully/qualified/path/
+}
+```
+
+### Volumes
+
+Marathon pods support several types of volumes are supported: ephemeral, persistent local and persistent external. All volumes are defined at the pod level. Your pod definition must include a `volumes` field that specifies at least the name of the volume and a `volumeMounts` field that specifies at least the name and mount path of the volume.
+
+```json
+{
+	"volumes": [
+		{
+			"name": "etc"
+		}
+	]
+}
+```
+
+```json
+{
+	"volumeMounts": [
+		{
+			"name": "env",
+			"mountPath": "/mnt/etc"
+		}
+	]
+}
+```
+
+### Containerizers
+
+Marathon pods support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/). The Mesos containerizer [allows you to specify alternative container images, such as Docker](http://mesos.apache.org/documentation/latest/container-image/). [Learn more about running Docker containers on Marathon](http://mesosphere.github.io/marathon/docs/native-docker.html).
+
+The following JSON specifies a Docker image for the pod:
+
+```json
+{  
+   "image":{  
+      "id":"mesosphere/marathon:latest",
+      "kind":"DOCKER",
+      "forcePull":false
+   }
+}
+```
+
+### Network
+
+# Create and Manage Pods
+
+Use the `/v2/pods/` endpoint to create and manage your pods. [See the full API spec](http://mesosphere.github.io/marathon/docs/generated/api.html#v2_pods).
+
+## Create
+
+```json
+ $ http POST <ip>:<port>/v2/pods mypod.json
+```
+
+Sample response:
+
+```json
+HTTP/1.1 201 Created
```

Comment:
I would remove
'''+HTTP/1.1 201 Created
 +Cache-Control: no-cache, no-store, must-revalidate
 +Content-Type: application/json; qs=2
 +Date: Tue, 18 Oct 2016 21:07:49 GMT
 +Expires: 0
 +Location: http://localhost:8080/v2/pods/simplepod2
 +Marathon-Deployment-Id: 8b591de8-6f2d-4036-870e-7571912d3522
 +Pragma: no-cache
 +Server: Jetty(9.3.z-SNAPSHOT)
 +Transfer-Encoding: chunked
 +X-Marathon-Leader: http://71c812cd6810:8080
'''
It doesn't add value.  it elongates the response and it is dependent on tool being used... it also shows information regarding dates and versions which will "feel" old a year from now regardless of the information still being accurate.